GitExtUtils/FileSystemUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+
+namespace GitExtUtils
+{
+    public static class FileSystemUtils
+    {
+        public static bool IsRootedDirectoryPath(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return false;
+                }
+
+                // this is going to throw if it's an invalid path (e.g. contains special chars)
+                var info = new DirectoryInfo(path);
+
+                return Path.IsPathRooted(path.Trim());
+            }
+            catch (Exception)
```

Comment:
Should this catch on all Exceptions or just the specific I/O exception?

Reply:
I got your point and in general it makes a lot of sense.

In this specific case, I'd prefer to use a more generic exception handler: I guess it's because it communicates the fact that, no matter what, this is not expected to throw any exception. And that if anything goes bad, "false" is the result. 

That said, I recognize it's probably more about having a more consistent code base (in terms of coding styles) so just let me know and I'll fix it! ;)

Reply:
The code style varies wildly...
A comment would be good.